examples/globe.js

```diff
@@ -36,7 +36,7 @@ if (!renderer) {
     miniView.mainLoop.gfxEngine.renderer.setClearColor(0x000000, 0);
 
     // update miniview's camera with the globeView's camera position
-    globeView.onAfterRender = function onAfterRender() {
+    globeView.onAfterRender.push(() => {
```

Comment:
Typically, this use case of `I want to execute something after each render` is a very good fit for a frame requester hooked on a `postRender` (or better in this case: `postUpdate`) phase.